lib/mwoffliner.lib.js

```diff
@@ -220,8 +220,36 @@ function execute(argv) {
                     parsoidConfig.setMwApi({
                         uri: '${mw.base + mw.apiPath}',
                     });
+
+                    parsoidConfig.timeouts = {
+
+                        // How long does a request have to generate a response?
+			request: 12 * 60 * 1000,
```

Comment:
12 mins seems very long! On the WMF cluster we use 2 mins, but in reality, p99 numbers are ~60 sec. But, not harmful to use 12 mins .. just noticing. Maybe start with 5 mins? In any case, I want to extract this into a separate config file as well and eventually add options to pass in other config files.

Reply:
Yes... I just try to find where things go wrong with #149 ... and in fact this PR seems to no 100% fix the problem. See my last comment there.